index.html

```diff
@@ -0,0 +1,309 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Home</title>
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <header class="toolbar">
+        <a href="./index.html" class="logo"
+          ><img src="./image/images-head/logo.png" alt="MyLogo" />
+        </a>
+        <nav class="nav-btn">
+          <img src="./image/images-head/union.png" alt="navbarimage" />
+        </nav>
+      </header>
+      <main>
+        <section class="headline">
+          <h1 class="headline-title">I'm Sylvestre Glad to see you!</h1>
+          <p class="headline-info">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <a href="#" class="media-info">LET'S CONNECT</a>
+          <ul class="social-icons">
+            <li>
+              <a href="#"
+                ><img src="./image/images-main/twitter.svg" alt="twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./image/images-main/linkedin.svg" alt="linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./image/images-main/medium.svg" alt="medium"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./image/images-main/github.svg" alt="github"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./image/images-main/angellist.svg" alt="angellist"
+              /></a>
+            </li>
+          </ul>
+        </section>
+        <section class="main-card-section">
+          <ul class="card-section">
+            <li class="card">
+              <div class="section-img">
+                <img
+                  src="./image/images-main/snapshootportfolio.png"
+                  alt="myproject"
+                />
+              </div>
+              <h2 class="title">Tonic</h2>
+              <ul class="info">
+                <li>canopy</li>
+                <li>
+                  <img src="./image/images-main/Counter.png" alt="counter" />
+                </li>
+                <li>back end dev</li>
+                <li>
+                  <img src="./image/images-main/Counter.png" alt="counter" />
+                </li>
+                <li>2015</li>
+              </ul>
+              <p class="section-info">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups require
+              </p>
+              <ul class="program-btns">
+                <li class="btn-ctn">
+                  <button type="submit" class="btn-html">html</button>
```

Comment:
- [ ] **OPTIONAL**
Great work wrapping these elements on `ul` and `li` tags. However, please notice that this is a simple list of items that represent the languages used to make this project in the portfolio. So, according to the [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) let's remove the `button` tag since it's not actually going to make an action. Please tell me what you think!

> Kindly remember to apply this note on the similar `<button>` tags' situations.

Reply:
I think you are correct. I have made the required corrections.